dev-scripts/install-site.sh

```diff
@@ -42,8 +42,10 @@ curl --silent --show-error --fail --output /dev/null http://varnish:8080/
 # Enable dev modules (see task dev:enable-dev-tools).
 MODULES=(devel field_ui purge_ui restui uuid_url views_ui dblog)
 
-if [[ $SKIP_CONTENT != "true" ]]; then
+if [[ $BNF_CONTENT != "true" ]]; then
   MODULES=("${MODULES[@]}" dpl_example_content)
+else
+  MODULES=("${MODULES[@]}" bnf_example_content)
```

Comment:
I would appreciate a few more thoughts/commentary here.

As I see it this approach prevents us from not installing any content during installation as you can either pass `--bnf-content` to get `bnf_example_content` or not to get `dpl_example_content`.

I wonder whether you considered an approach where we support all three options with `--dpl-content`, `--bnf-content` or an argument `--content=bnf`, `--content=dpl` etc.

I recognize that we probably do not use the `--no-content` option at all at the moment though.

Reply:
> I recognize that we probably do not use the --no-content option at all at the moment though.

Exactly my thinking. `--no-content` was added because BNF needed to avoid the standard content, which was mostly because it made it rather confusing when you had two sites that were initially identical.